gc/gc-repository-jdbc/src/main/java/org/projectnessie/gc/contents/jdbc/JdbcPersistenceSpi.java

```diff
@@ -419,6 +429,16 @@ LiveContentSet liveContentSet(ResultSet rs) throws SQLException {
         .build();
   }
 
+  protected String decorateInsertStatement(String statement) {
+    switch (productName()) {
+      case "mysql":
+      case "mariadb":
+        return statement.replace("INSERT", "INSERT IGNORE");
+      default:
```

Comment:
Can you replace this with `case "postgres":` and let the `default:` case just throw?